packages/icons/icons.json

```diff
@@ -4527,5 +4527,19 @@
         "tags": "rotate cursor, curved, arrows",
         "group": "interface",
         "codepoint": 62343
+    },
+    {
+        "displayName": "Square: rotate clockwise",
```

Comment:
Why the "square:" before these? I feel like just "Rotate {counter}clockwise" would be sufficient.

Reply:
I feel like there is a reasonable argument for future-proofing the naming here in case we want to support a more generic "rotate" icon in the future, something like this:

![Screenshot 2024-11-04 at 15 22 36@2x](https://github.com/user-attachments/assets/7af939d9-cbae-4015-a588-a8dbdea17d6f)

The more descriptive (and less subjective) the naming conventions that we apply, the less likely that we'll run into naming collisions down the line as we continue to add more icons over time. To illustrate this, here's what a search for "rotate" turns up in other popular icon libraries:
- https://fontawesome.com/search?q=rotate&o=r
- https://mui.com/material-ui/material-icons/?query=rotate
- https://feathericons.com/?query=rotate
- https://icons.getbootstrap.com/?q=rotate
- https://www.streamlinehq.com/?search=rotate
- https://lucide.dev/icons/?focus=&search=rotate
- https://react-icons.github.io/react-icons/search/#q=rotate
- https://icon-sets.iconify.design/?query=rotate

Reply:
Reverted naming changes to ""Rotate {counter}clockwise" in 79484d984a52ea6df157523dec55868a6c87c559